src/main/kotlin/org/rust/ide/formatter/RustBlock.kt

```diff
@@ -0,0 +1,92 @@
+package org.rust.ide.formatter
+
+import com.intellij.formatting.*
+import com.intellij.lang.ASTNode
+import com.intellij.psi.TokenType.WHITE_SPACE
+import com.intellij.psi.tree.TokenSet
+import org.rust.lang.core.psi.RustCompositeElementTypes.*
+import org.rust.lang.core.psi.RustTokenElementTypes.*
+
+class RustBlock(
```

Comment:
I thought about naming this `RustFmtBlock`, but I don't think this will collide with PSI `RustBlock` anywhere apart from IntelliJ's Go To Class.